projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/provider/description/interestrate/NormalSTIRFuturesSmileProvider.java

```diff
@@ -20,7 +20,7 @@
  * Implementation of a provider of normal volatility (Bachelier model) smile for options on STIR futures. The volatility is time to expiration/strike/delay dependent.
```

Comment:
Change the Javadoc to
"Implementation of a provider of normal volatility (Bachelier model) smile for options on STIR futures. The volatility is time to expiration/strike dependent."